SparkPost/2web2ui
# FE-1094 - Ensure CircleCI badge renders properly

[FE-1094](https://jira.int.messagesystems.com/browse/FE-1094)

### What Changed
- Fixed status badge markup by incorporating API token - see https://discuss.circleci.com/t/github-badges-broken/3706/7
- Removes Travis CI badge

### How To Test
- Look at our Readme rendering in GitHub by swapping to the FE-1094 branch in the UI. Should see the rendered badge (which should reflect the build status for `master`)
